packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+### Toegankelijkheidseisen
 
-- [Terugdringen van onnodige vragen naar sekse](https://www.denhaag.nl/nl/bestuur-en-organisatie/contact-met-de-gemeente/terugdringen-van-onnodige-vragen-naar-sekse.htm)
+Formulierelementen moeten aan de volgende eisen voldoen, ze hebben:
 
-## Accessibility
+* Een label of instructie die is gekoppeld aan het formulierelement,
+* Bij een fout een duidelijke foutmelding,
+* Bij een foutmelding moet duidelijk worden uitgelegd hoe deze fout kan worden verholpen en hoe deze in de toekomst kan worden voorkomen.
+* Als een invoer een fout genereert, zorg er dan voor dat de bijbehorende foutmelding op het juiste moment en op de juiste manier door voorleessoftware wordt uitgesproken.
 
-Form elements must meet the following requirements, they have:
+## Formulier interactie; vragen en antwoorden
 
-1. A label or instruction associated with the form element,
-2. In case of an error, a clear error message,
-3. In the event of an error message, a clear explanation must be given on how to correct this error and how to prevent it in the future.
+Vraag alleen om de strikt noodzakelijke informatie en informeer gebruikers over de noodzaak van de gevraagde informatie.
 
-If an input generates an error, make sure that the corresponding error message is announced by screen readers at the right time and in the right way.
+Bij iedere vraag dien je je af te vragen of deze daadwerkelijk nodig is voor de bezoeker en/of organisatie. Is informatie al binnen de organisatie aanwezig, vraag je dan af of deze nogmaals gevraagd dient te worden, bv. ter controle of ter wijziging door de bezoeker.
+
+Indien het onduidelijk is waarom informatie gevraagd wordt informeer bezoekers dan over de noodzaak hiervan maar ook wat de voordelen voor hen zijn.
+
+Voorkom dat gebruikers dezelfde informatie meerdere keren moeten invoeren.
+
+### Tekst labels
+
+Labels zijn de vragen die je een gebruiker stelt. Hier ga je een dialoog aan, dus zorg dat deze zo duidelijk en vriendelijk maar tevens beknopt mogelijk zijn. Laat een bezoeker de vragen makkelijk scannen.
+
+Een goed label wordt buiten een invoerveld geplaatst, niet daarbinnen (de zg. _‘floating labels’_). Op deze manier heeft het label altijd een vaste positie en kan het in een leesbare grootte worden weergegeven. Een label dat in een invoerveld is geplaatst, kan een bezoeker in verwarring brengen, aangezien het voor reeds ingevoerde tekst aangezien kan worden. De leesbaarheid lijdt er ook onder omdat het contrast tussen tekst voorgrond- en achtergrondkleur vaak te laag is. Het kan ook verward worden met inactieve of placeholder tekst. Bovendien kan het ook nodig zijn om soms daadwerkelijke placeholder tekst te tonen.
```

Comment:
Zouden we hier ook iets moeten zeggen over de hybride variant waar label als placeholder on focus de label positie inneemt? En dat het verplaatsen van de label vanuit de placeholder positie naar de label positie naast de onduidelijkheden hierboven beschreven ook voor screen readers super onduidelijk is?

Reply:
Ik denk dat dit slim is om te doen. Placeholders alleen gebruiken als het echt functioneel iets toevoegt om een patroon van invulling weer te geven maar geef dan ook in het veld het maximum aantal karakters aan.

Reply:
@Yolijn Bedoel je met de hybride variant waar het label de placeholder positie inneemt totdat focus op het veld staat het [floating label patroon](https://material.io/components/text-fields)? Als het even kan zou ik die inderdaad willen ontmoedigen vanwege de vele usability en accessibility problemen die hier aan kleven.